dist/PathTools/t/Spec.t

```diff
@@ -1,6 +1,5 @@
 #!/usr/bin/perl -w
 
-use strict;
```

Comment:
we should preserve that `use strict` in dual life modules... the module is going to be installed using older versions of Perl
we could even consider the reverse, by enforcing `use strict; use warnings` on all these tests and files...

Reply:
Noted, see commit messages in updated p.r.